tests/chainerx_tests/unit_tests/routines_tests/test_logic.py

```diff
@@ -104,24 +117,35 @@ def check(x, y):
     check(b, a)
 
 
-@chainerx.testing.numpy_chainerx_array_equal()
-@pytest.mark.parametrize('a_object', [
-    ([]),
-    ([0]),
-    ([1]),
-    ([0.2]),
-    ([-0.3]),
-    ([True]),
-    ([False]),
-    ([0, 1, 2]),
-    ([0., numpy.nan]),
-    ([numpy.nan, numpy.inf]),
-    ([-numpy.inf, numpy.nan]),
-    ([[0, 1], [2, 0]]),
+@op_utils.op_test(['native:0', 'cuda:0'])
+@chainer.testing.parameterize_pytest('a_object', [
+    ([],),
+    ([0],),
+    ([1],),
+    ([0.2],),
+    ([-0.3],),
+    ([True],),
+    ([False],),
+    ([0, 1, 2],),
+    ([0., numpy.nan],),
+    ([numpy.nan, numpy.inf],),
+    ([-numpy.inf, numpy.nan],),
+    ([[0, 1], [2, 0]],),
```

Comment:
Is it tricky to align the behavior of `chainer.testing.parameterize_pytest` with `pytest.mark.parametrize`? It's probably outside the scope of this PR so I think it's okay to leave it as is, but for future use-cases.

Reply:
Fixed.